common/v2/features/Dashboard/components/RecentTransactionList.tsx

```diff
@@ -200,15 +200,21 @@ export default function RecentTransactionList({ accountsList, className = '' }:
   const filteredGroups = groups.filter(group => group.entries.length !== 0);
 
   const recentTransactionsTable = {
-    head: ['Date', 'From Address', 'To Address', 'Amount', 'View More'],
+    head: [
+      translateRaw('RECENT_TRANSACTIONS_DATE') || 'Date',
+      translateRaw('RECENT_TRANSACTIONS_FROM_ADDRESS'),
+      translateRaw('RECENT_TRANSACTIONS_TO_ADDRESS'),
+      translateRaw('RECENT_TRANSACTIONS_TO_AMOUNT'),
+      translateRaw('RECENT_TRANSACTIONS_VIEW_MORE') || 'View More'
```

Comment:
Why do we need the `|| 'View More'` here?

Reply:
Well in case, there is no translation available I wanted to default to some string. Because, those columns are used in sortable columns, and in hidden columns and providing empty string, would break the table. But I am wondering, if this is necessary since. Since it will default to english translation anyway. Should I remove it?

Reply:
> Since it will default to english translation anyway. Should I remove it?

I'd say so c: